compact/ecal/bic/bic.xml

```diff
@@ -131,9 +138,17 @@
       calorimeterType="EM_BARREL"
       vis="EcalBarrelEnvelopeVis"
       offset="EcalBarrel_Calorimeter_offset">
+      <type_flags type="DetType_TRACKER + DetType_CALORIMETER + DetType_BARREL"/>
       <dimensions numsides="EcalBarrel_SectorRepeat"
         rmin="EcalBarrel_rmin"
         z="EcalBarrel_Calorimeter_length"/>
+      <envelope vis="EcalBarrelImaging_envelope"
+        rmin="EcalBarrel_rmin"
+        rmax="EcalBarrel_rmin + 2.0 * EcalBarrel_ImagingLayerThickness"
```

Comment:
```suggestion
        rmax="EcalBarrel_rmin + EcalBarrel_FrontSupportThickness + EcalBarrel_ImagingLayerThickness"
```
Note: this will keep the first SFIL out of the envelope but likely won't contain the AstroPix when using the staves mounted to the 'roof' of the tray.